pkg/ibm/vpc_collection.go

```diff
@@ -58,7 +58,13 @@ func getVPCs(vpcService *vpcv1.VpcV1, region, resourceGroupID string) ([]*datamo
 	}
 	res := make([]*datamodel.VPC, len(vpcs))
 	for i := range vpcs {
-		res[i] = datamodel.NewVPC(&vpcs[i], region)
+		listVpcAddressPrefixesOptions := &vpcv1.ListVPCAddressPrefixesOptions{}
+		listVpcAddressPrefixesOptions.SetVPCID(*vpcs[i].ID)
+		addressPrefixes, _, err := vpcService.ListVPCAddressPrefixes(listVpcAddressPrefixesOptions)
```

Comment:
Better use `iteratePagedAPI` to support the (unlikely) case where a VPC has more than 50 address prefixes. Just to be sure...